DSL2: Dev hostremoval (#965)
nf-core/eager: docs/output.md

  [BWA](https://bio-bwa.sourceforge.net/bwa.shtml) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome. BWA-backtrack (a.k.a `bwa aln`) is designed for Illumina sequence reads up to 100bp, while BWA-MEM (`bwa mem`) is optimised for longer sequences ranged from 70bp to 1Mbp and split alignment.

### Host Removal

<details markdown="1">
<summary>Output files</summary>

- `host_removal/`

  - `*.fq.gz`: Fastq files containing only reads that did not map
    to the reference genome.

</details>

> nf-core/eager host removal step removes any reads from the original input fastq files that mapped to the provided reference. This step is performed by an a costume python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).

Reviewer: ```suggestion
nf-core/eager host removal step removes any reads from the original input fastq files that mapped to the provided reference. This step is performed by a custom python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).
```

> The resulting fastq files can be used for publication purposes where
> the host DNA should not be made available for ethical reasons (for example when you do not have permission to analyse the host DNA but analysis on the metagenomic content were agreed) or for due to identifiability reasons (for example, for individualsthat died in recent times and could be identified based on their DNA alone).

Reviewer: ```suggestion
The resulting FASTQ files can be used for publication purposes where
the host DNA should not be made available for ethical reasons (for example when you do not have permission to analyse the host DNA but only analysis on the metagenomic content were agreed) or for due to identifiability reasons (for example, for individuals that died in recent times and could be identified based on their DNA alone).

Alternatively you could use it for manual metagenomic screen outside of nf-core/eager (e.g. when your preferred taxonomic classifier isn't supported by nf-core/eager).
```

  [Bowtie 2](https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml) is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s of characters to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index (based on the Burrows-Wheeler Transform or BWT) to keep its memory footprint small and supports gapped, local, and paired-end alignment modes.

### Host Removal

<details markdown="1">
<summary>Output files</summary>

- `host_removal/`

  - `*.fq.gz`: FASTQ files containing only reads that did not map
    to the reference genome.

</details>

> nf-core/eager host removal step removes any reads from the original input FASTQ files that mapped to the provided reference. This step is performed by a custom python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).

Reviewer: ```suggestion
The nf-core/eager host removal step removes any reads from the original input FASTQ files that mapped to the provided reference. This step is performed by a custom python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).
```

  [Bowtie 2](https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml) is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s of characters to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index (based on the Burrows-Wheeler Transform or BWT) to keep its memory footprint small and supports gapped, local, and paired-end alignment modes.

### Host Removal

<details markdown="1">
<summary>Output files</summary>

- `host_removal/`

>   - `*.fq.gz`: Fastq files containing only reads that did not map

Reviewer: ```suggestion
  - `*.fq.gz`: FASTQ files containing only reads that did not map
```

    to the reference genome.

</details>

> nf-core/eager host removal step removes any reads from the original input fastq files that mapped to the provided reference. This step is performed by a custom python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).

Reviewer: ```suggestion
nf-core/eager host removal step removes any reads from the original input FASTQ files that mapped to the provided reference. This step is performed by a custom python script [extract_map_reads.py](https://github.com/nf-core/eager/blob/master/bin/extract_map_reads.py).
```

> The resulting FASTQ files can be used for publication purposes where
> the host DNA should not be made available for ethical reasons (for example when you do not have permission to analyse the host DNA but only analysis on the metagenomic content were agreed) or for due to identifiability reasons (for example, for individuals that died in recent times and could be identified based on their DNA alone).

Reviewer: ```suggestion
The resulting FASTQ files can be used for publication purposes where the host DNA should not be made available for ethical reasons (for example when you do not have permission to analyse the host DNA but only analysis on the metagenomic content were agreed) or for due to identifiability reasons (for example, for individuals that died in recent times and could be identified based on their DNA alone).
```
